doc/sphinx/source/install.rst

```diff
@@ -389,3 +389,41 @@ RISC-V specific build notes:
 
   from linker, then most likely PoCL used the incorrect ABI. You can explicitly
   specify the ABI to use with the HOST_CPU_TARGET_ABI CMake option.
+
+PowerPC support
```

Comment:
This reminds me: Can you add a table to the README.md which lists the known working processors/OSs and their known maturity levels? It would be nice to have such a summary so people know what to expect. The levels could be experimental/usable/mature/conformant or similar. I'd say CUDA=usable although it's experimental, for example.